mlflow/ml-package-versions.yml

```diff
@@ -255,6 +255,12 @@ fastai:
     minimum: "2.4.1"
     maximum: "2.7.17"
     requirements:
+      # TODO: fastai depends on spacy, but spacy>=3.8.2 requires python>=3.9
+      #  Once migrating fastai tests to python 3.9, we should remove spacy<3.8.2
+      #  To migrate fastai tests to python 3.9, we need to address error
+      #  when installing old version scipy: "No module named 'distutils.msvccompiler'"
+      #  similar issue link: https://github.com/numpy/numpy/issues/27405
```

Comment:
this needs to pin setuptools to old version, which requires to modify CI python setup code, can be done in follow-up PR.